### PR TITLE
Phase1 development

### PR DESCRIPTION
**Pull request by shinigami22**
migrations for poets,poems and poet_informations done

### CODE REVIEW

File: Gemfile [hunk 1]
```diff
@@ -40,6 +40,6 @@ group :development, :test do
   gem 'web-console', '~> 2.0'
 
   # Spring speeds up development by keeping your application running in the background. Read more: https://github.com/rails/spring
-  gem 'spring'
+  gem 'spring', '~>1.3.3'
```
Comment (devudilip) [inline Gemfile:43]: Why did you change spring version ?

File: db/migrate/20150312062424_create_poets.rb [hunk 2]
```diff
@@ -0,0 +1,14 @@
...
+      t.string "name", :limit => 45
+      t.string "sex", :limit => 45
+      t.timestamps null: false
+    end
+     add_index(:poets,:id)
+  end
+  def down
+  	 drop_table :poets
+  end
+end
```
Comment (devudilip) [inline db/migrate/20150312062424_create_poets.rb:14]: Please maintain alignment
(Ruby alignment rules)

File: db/migrate/20150312062424_create_poets.rb [hunk 3]
```diff
@@ -0,0 +1,14 @@
+class CreatePoets < ActiveRecord::Migration
+  def up
+    create_table :poets do |t|
+      t.string "pen_name", :limit => 45
+      t.string "name", :limit => 45
+      t.string "sex", :limit => 45
+      t.timestamps null: false
```
Comment (devudilip) [inline db/migrate/20150312062424_create_poets.rb]: please take out null: false.

update_at can be null for new objects. So with your change we will end up with an error later.

File: db/migrate/20150312062424_create_poets.rb [hunk 4]
```diff
@@ -0,0 +1,14 @@
...
+    end
+     add_index(:poets,:id)
+  end
```
Comment (devudilip) [inline db/migrate/20150312062424_create_poets.rb:10]: remove index 
id is default

File: db/migrate/20150312062424_create_poets.rb [hunk 5]
```diff
@@ -0,0 +1,14 @@
+class CreatePoets < ActiveRecord::Migration
+  def up
```
Comment (devudilip) [inline db/migrate/20150312062424_create_poets.rb:2]: please remove all limits

File: test/fixtures/poems.yml [hunk 6]
```diff
@@ -0,0 +1,11 @@
+# Read about fixtures at http://api.rubyonrails.org/classes/ActiveRecord/FixtureSet.html
```
Comment (devudilip) [inline test/fixtures/poems.yml:1]: Do we really need this file ?

Please dont push any unwanted files.

File: test/models/poem_test.rb [hunk 7]
```diff
@@ -0,0 +1,7 @@
+require 'test_helper'
```
Comment (devudilip) [inline test/models/poem_test.rb:1]: We will be using Rspec so u can neglect this files.

Whatever from test/\* u can remove

File: Gemfile (hunk 1 above)
Comment (shinigami22) [inline Gemfile:43]: i got an error when i tried generating model. "spring 1.3.2 is use, but
spring 1.3.3 is active"
